src/dataproxy/parent/ParentWindowProvider.tsx

```diff
@@ -30,7 +30,16 @@ export const ParentWindowProvider = React.memo(
       }
     }
 
+    const sendReadyMessage = (): void => {
+      // This is useful to inform the parent apps that the DataProxy's iframe is ready to be requested
+      window.parent.postMessage(
+        { type: 'DATAPROXYMESSAGE', payload: 'READY' },
+        '*'
+      )
+    }
+
```

Comment:
nit: this can be easily moved outside of the react component for better readability.